capybara/selenium/driver.py

```diff
@@ -49,8 +53,11 @@ def browser(self):
             capabilities = (capabilities or DesiredCapabilities.FIREFOX).copy()
             # Auto-accept unload alerts triggered by navigating away.
             capabilities["unexpectedAlertBehaviour"] = "ignore"
+        elif self._browser_name == "chrome":
+            capabilities = (capabilities or DesiredCapabilities.CHROME).copy()
```

Comment:
What's this for? It doesn't appear to be connected to the addition of `config_obj`. And [`webdriver.Chrome` already does this](https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/chrome/webdriver.py#L50) when no `desired_capabilities` are provided.